src/Controller/CategoriesController.php

```diff
@@ -41,10 +41,11 @@ public static function create(ContainerInterface $container) {
   /**
    * Provides a list of categories uuid's that contains videos.
```

Comment:
Would be good to update the function description and param info

Reply:
Fixed